mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -141,6 +135,48 @@ def __init__(self, name=None, bus=None, use_settings=True):
         self.bind(bus)
         #: Mycroft global configuration. (dict)
         self.config_core = Configuration.get()
+
+        self.settings = None
+
+        #: To not break existing setups,
+        #: save to skill directory if the file exists already
+        self.settings_write_path = Path(self.root_dir)
+
+        #: Otherwise save to XDG_CONFIG_DIR
+        if not self.settings_write_path.joinpath('settings.json').exists():
+            self.settings_write_path = Path(BaseDirectory.save_config_path(
+                'mycroft', 'skills', self.name))
+
+        self.settings_read_path = None
+
+        #: To note break existing setups,
```

Comment:
note -> not

Reply:
Also the `#:` comments are things that will be picked up by the [sphinxdocs](https://mycroft-core.readthedocs.io/en/latest/) and these should be more on how to use the member variables, this is probably a note to core developers and not skill developers so the `:` can be removed. (goes for a bunch of other places as well)